jira/client.py

```diff
@@ -196,7 +197,7 @@ class JIRA(object):
     AGILE_BASE_URL = GreenHopperResource.AGILE_BASE_URL
 
     def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=None, kerberos=False,
-                 validate=False, get_server_info=True, async=False, logging=True, max_retries=3, proxies=None):
+                 validate=False, get_server_info=True, async=False, max_retries=3, proxies=None, **kwargs):
```

Comment:
Serious objection to this approach.

This allows callers to continue supplying a `logging` argument and have it silently ignored (which is itself a breaking change of the API). If you are arguing for no breaking change in this revision then you have to continue to support the logging argument and its current behaviour. (I have previously tried to argue that the current behaviour of this undocumented argument is of little value and likely not of use to anyone.)

Additional objection to adding `**kwargs` is that this will allow any other argument to be accepted and silently ignored by the constructor. This may include arguments that don't exist, or that are typos of arguments that do exist. e.g. `get_serverinfo=False`. This is terrible for usability of this API.

If you don't care about preserving the behaviour associated with the `logging` argument, but just want to make sure that no code breaks due to syntax errors, then leave logging as an explicit argument, and document it as unsupported in the docstring. (and hopefully remove it in the next major revision?)